aws-s3/aws.ml

```diff
@@ -191,6 +192,6 @@ module Make(Io : Types.Io) = struct
       (Headers.add_opt ~key:"Authorization" ~value:auth headers), body
     in
     body >>= fun body ->
-    Http.call ~domain ~scheme ~host:host_str ~path ~query ~headers ~expect ~sink ?body meth >>=? fun (code, msg, headers, body) ->
+    Http.call ?port ~domain ~scheme ~host:host_str ~path ~query ~headers ~expect ~sink ?body meth >>=? fun (code, msg, headers, body) ->
```

Comment:
It makes sense to be able to specify the port number here. I actually think it should be mandatory and not optional.